tests/Validator.js

```diff
@@ -530,6 +530,23 @@ class Validator {
             // do nothing
         }
     }
+
+    /**
+     * VehicleDataResult equals validation.
+     * @param {VehicleDataResult} item1 - A VehicleDataResult.
+     * @param {VehicleDataResult} item2 - A VehicleDataResult.
+     * @returns {Boolean} - Whether or not they're equal.
+     */
+    static validateVehicleDataResult (item1, item2) {
+        if (item1 === null || item2 === null) {
+            expect(item1).to.equal(item2);
+            return true;
+        }
+        expect(item1.getDataType()).to.be.equal(item2.getDataType());
```

Comment:
When comparing the two items' datatype, oemcustomdatatype, and resultcode does a null check need to be done to ensure that the datatype, oemcustomdatatype, or resultcode is not null?

Reply:
this is done the same way as other validators, eg. `validateTemplateColorScheme` in this file.

Reply:
ok sounds good.  Thanks.